src/applications/ivc-champva/10-7959f-1/config/submitTransformer.js

```diff
@@ -30,6 +30,14 @@ export default function transformForSubmit(formConfig, form) {
     },
     // statement_of_truth_signature: transformedData.fullName,
     current_date: new Date().toJSON().slice(0, 10),
+    primaryCustomer: {
```

Comment:
I think this should be `primaryContactInfo` rather than customer - otherwise, looks good to me